tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/quickaccess/QuickAccessProvidersTest.java

```diff
@@ -0,0 +1,130 @@
+package org.eclipse.ui.tests.quickaccess;
```

Comment:
Raghu, the test should be added to the test suite.

Reply:
Added to QuickAccess Testsuite.